src/test/java/org/jenkinsci/plugins/secure_requester_whitelist/SecureRequesterImplTest.java

```diff
@@ -42,7 +42,10 @@
     @Test public void authorizing() throws Exception {
         assertJSONP(null, 403);
         assertJSONP("http://apache.org/", 403);
-        Whitelist.get().configure(null, new JSONObject().accumulate("allowNoReferer", true).accumulate("domains", "apache.org jenkins-ci.org"));
+        final Whitelist whitelist = Whitelist.get();
```

Comment:
Why is this change necessary?

Reply:
The old code sent in a json object but a `null` `StaplerRequest` the new code needs it to do the binding.